### ISSUE TITLE
DISCOVERY: Accessibility Standards and Compliance

### ISSUE THREAD

**Issue by awilcoxa** [magento/pwa-studio#982]
Goal: Complete a research spike to identify testing and analysis tools/process in order to identify standards and current gaps in accessibility requirements. Result of this research item should generate backlog issues for prioritization and grooming.

Open questions: 
What is the accessibility requirement(s) for  Peregrine vs Venia?

**Comment by awilcoxa** [comment]
consider impact of tab ordering as a part of this discovery effort

**Comment by soumya-ashok** [comment]
Tab order should probably be linked to this ticket @awilcoxa since that is related to accessibility.

**Comment by soumya-ashok** [comment]
Comprehensive list of things to meet WCAG - https://www.w3.org/WAI/WCAG21/quickref/

This would probably need to be broken down into granular tasks, and prioritized to be phased into the storefront / components.

**Comment by ericerway** [comment]
@awilcoxa @soumya-ashok Would it be worthwhile starting with Lighthouse considerations for Accessibility first on a page/route by page/route basis? We'll want to perform a formal audit with Accessibility in the future once we have the basics covered.

**Comment by awilcoxa** [comment]
> @awilcoxa @soumya-ashok Would it be worthwhile starting with Lighthouse considerations for Accessibility first on a page/route by page/route basis? We'll want to perform a formal audit with Accessibility in the future once we have the basics covered.

This was already worked on and merged in PR #1395. If we discover other lighthouse driven issues those of course go right into grooming process. This discovery issue is meant to address larger accessibility compliance standards for future Venia implementation.

**Comment by awilcoxa** [comment]
Probably fair to state that our target for WCAG 2.1 compliance should be AA level given that it's specifically required by EU Directive (which seems to be the most strict standard that is currently set). 

Here's the list of AA requirements: https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=a%2Caaa#top 

@soumya-ashok let's work to build out a chart displaying where we currently meet these standards vs don't. That will give us our requirements for upcoming accessibility effort.

**Comment by awilcoxa** [comment]
We have established clear next steps for requirements gathering and will continue to track additional discovery/research through internal Confluence page: https://wiki.corp.magento.com/display/FF/PWA+Studio+Accessibility+Research

Will publish results in to backlog and dev docs as we move forward.
